var/spack/repos/builtin/packages/globalarrays/package.py

```diff
@@ -25,42 +25,59 @@
 from spack import *
 
 
-class Globalarrays(CMakePackage):
-    """The Global Arrays (GA) toolkit provides a shared memory style
-    programming environment in the context of distributed array data
-    structures.
-    """
+class Globalarrays(AutotoolsPackage):
+    """Global Arrays (GA) is a Partitioned Global Address Space (PGAS)
+    programming model. It provides primitives for one-sided communication
+    (Get, Put, Accumulate) and Atomic Operations (read increment).
+    It supports blocking and non-blocking primtives, and supports location
+    consistency."""
 
     homepage = "http://hpc.pnl.gov/globalarrays/"
-    url = "https://github.com/GlobalArrays/ga"
+    url = "https://github.com/GlobalArrays/ga/releases/download/v5.7/ga-5.7.tar.gz"
 
-    version('master', git='https://github.com/GlobalArrays/ga', branch='master')
+    version('5.7',   'bb9a441a6b4fbb8b52b58c2d3f4cd07f')
+    version('5.6.5', '90da628dc72048deeda3f0cd095cb5b3')
+    version('5.6.4', '051901b316c9766b8ba54306bff7f6b3')
+    version('5.6.3', 'df3cf6cc8288d9f202b7fd0ea82f5491')
+    version('5.6.2', '901e4612203bac45059be524fc1abfb7')
+    version('5.6.1', '674c0ea9bf413840b1ff1e669de73fca')
+    version('5.6',   '49d7e997daed094eeb9565423879ba36')
 
-    variant('i8', default=False, description='Build with 8 byte integers')
+    variant('bytes', values=('4', '8'), default='8', 
```

Comment:
please make it a `bool` and call `int64` to be consistent with many other packages in Spack.

Reply:
To be sure I'm following, something like this?

`variant('bytes', default=True, description='Build with 8 byte integers')`

I'm not quite understanding what you mean by calling `int64`?

Reply:
> I'm not quite understanding what you mean by calling int64?

`variant('int64', default=False, description='Compile with 64 bit indices support')`

Reply:
Ah, that makes sense. I can make that change.

Reply:
thanks! 👍

Reply:
New commit now includes the bytes > int64 change